tests/kubernetes.py

```diff
@@ -7,7 +7,9 @@ def load_kube_config(self, *args, **kwargs):
 
 
 class MockedKubernetesClient():
-    def __init__(self, jobs=[]):
+    def __init__(self, jobs=None):
```

Comment:
it is basically the same but more lines of code. What is benefit ?

Reply:
> it is basically the same but more lines of code. What is benefit ?

Using mutable objects such as lists and dicts for default values in function definition is dangerous and could have side-effects.  It's common Python gotcha as documented [here](https://docs.python-guide.org/writing/gotchas/#mutable-default-arguments).  Consider the following code:

```
def f(array=[]):
    array.append(1)
    return array

def f2(array=None):
    if array is None:
        array = []
    array.append(1)
    return array

print(f())
print(f())

print(f2())
print(f2())
```

It prints:
```
[1]
[1, 1]
[1]
[1]
```